Provide example network policies for the web terminal operator (#78504)
openshift/openshift-docs: web_console/web_terminal/troubleshooting-web-terminal.adoc

[id="troubleshooting-web-terminal-network-policies"]
== Web terminal and network policies

> The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state until a context deadline exceeded error occurs.

Reviewer: Is there a way to reformat "context deadline exceed" to let readers know it is a type of error? Maybe something like "...until a "context deadline exceeded" error occurs" or "...until a `context deadline exceeded` error occurs".

Author: Sounds good, I've updated to use the latter:
![image](https://github.com/user-attachments/assets/264d19b3-e240-4ffa-8179-ae139a594c78)

> The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state until a `context deadline exceeded error` occurs.

Reviewer 2: IBM Style word usage advises against using the terms _launch_ and _appears_. Consider a revision like this one:
```suggestion
The web terminal might fail to start if the cluster has network policies configured. To start a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to start and the terminal panel is in a loading state until a `context deadline exceeded error` occurs.
```

Author: Thank you, the suggestion looks great, I've applied it in my latest force push

To avoid this issue, ensure that the network policies for namespaces that are used for terminals allow ingress from the `openshift-console` and `openshift-operators` namespaces.

The following samples show `NetworkPolicy` objects for allowing ingress from the `openshift-console` and `openshift-operators` namespaces.

> .Example network policy for allowing ingress from openshift-console`

Reviewer 3: ```suggestion
.Example network policy for allowing ingress from the `openshift-console` namespace
```

> .Example network policy for allowing ingress from the `openshift-console` namespace

Reviewer: Can these headings be shortened? Standard headings are very brief ("Example input", etc.).

Author: Updated to:
```
Allowing ingress from the `openshift-console` namespace
```

WDYT?

[source,yaml]
----
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: allow-from-openshift-console
spec:
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          kubernetes.io/metadata.name: openshift-console
  podSelector: {}
  policyTypes:
  - Ingress
----

> .Example network policy for allowing ingress from openshift-operators

Reviewer 3: ```suggestion
.Example network policy for allowing ingress from the `openshift-operators` namespace
```

> .Example network policy for allowing ingress from the `openshift-operators` namespace

Reviewer: Similar note regarding shortening the headings, if possible.

Author: Updated to:
```
Allowing ingress from the `openshift-operators` namespace
```

WDYT?
